lib/jekyll-redirect-from/redirectable.rb

```diff
@@ -0,0 +1,25 @@
+module JekyllRedirectFrom
+  # Module which can be mixed in to documents (and pages) to provide
+  # redirect_to and redirect_from helpers
+  module Redirectable
+    # Returns a string representing the relative path or URL
+    # to which the document should be redirected
+    def redirect_to
+      if to_liquid["redirect_to"].is_a?(Array)
+        to_liquid["redirect_to"].compact.first
```

Comment:
Correct. `redirect_to` *can* be an array, an we absorb the complexity of YAML and just deal with it. I'm fine with that, as many users think that array-style markup with a single entry is the proper way to do things (and I'd rather add a few method calls then break their site).